probe/cobra.go

```diff
@@ -0,0 +1,35 @@
+package probe
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func CobraCommand() *cobra.Command {
+	var (
+		probeOptions Options
+	)
+
+	prb := &cobra.Command{
+		Use:   "probe",
+		Short: "Check the liveness or readiness of a locally-running server",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			if !probeOptions.IsValid() {
+				return fmt.Errorf("some options are not valid")
+			}
+			if err := NewFileClient(&probeOptions).GetStatus(); err != nil {
+				return fmt.Errorf("fail on inspecting path %s: %v", probeOptions.Path, err)
+			}
+			fmt.Fprintf(cmd.OutOrStdout(), "OK")
+			return nil
+		},
+	}
+	prb.PersistentFlags().StringVar(&probeOptions.Path, "probe-path", "",
+		"Path of the file for checking the availability.")
+	prb.PersistentFlags().DurationVar(&probeOptions.UpdateInterval, "interval", 0,
+		"Duration used for checking the target file's last modified time.")
```

Comment:
Can you rewrite this or ask for someone from @istio/wg-docs-maintainers to take a look.  I don't understand what this means.  Will `mixer probe` fail if the file is older than the interval?

Reply:
Yes, you are correct. `probe` checks the last modified time and it fails if it is older than the interval in [GetStatus()](https://github.com/istio/pkg/blob/master/probe/client.go#L40). It may be needless to say, but server side like `mixer server` updates the file's timestamp.

IMO, it would be better to add more detail in the help description so I added it https://github.com/istio/pkg/pull/97/files#diff-d50c947d5e646a9af72cb90ca19920ccR12-R13 . If I should put more detail in the option, please let me know (I am pinging docs team in docs channel in the Slack to take a look as well.)